### PR TITLE
Currency: created build script and build_info.json for Multidict

### PR DESCRIPTION
**Pull request by aastha-sharma2**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Have you validated script on UBI 9 container
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Review by shubham-dayma-ibm — APPROVED** [review]
LGTM

File: m/multidict/multidict_ubi_9.3.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+git clone $PACKAGE_URL
+cd $PACKAGE_NAME  # Change directory to the cloned repository
+git checkout $PACKAGE_VERSION  # Checkout the specified version
+
+# install necessary Python packages
+pip install .
+pip install pytest pytest-cov build
+
+#build
+if ! (pyproject-build) ; then
```
Comment (shubham-dayma-ibm) [inline m/multidict/multidict_ubi_9.3.sh:39]: Is this creating the wheel? If yes why we need to build the package?

File: m/multidict/multidict_ubi_9.3.sh [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+# Install dependencies
+yum install -y git gcc gcc-c++ make wget openssl-devel bzip2-devel libffi-devel zlib-devel python-devel python-pip cmake
+
+# Clone the repository
+git clone $PACKAGE_URL
+cd $PACKAGE_NAME  # Change directory to the cloned repository
+git checkout $PACKAGE_VERSION  # Checkout the specified version
+
+# install necessary Python packages
+pip install .
```
Comment (shubham-dayma-ibm) [inline m/multidict/multidict_ubi_9.3.sh:35]: Don't see a case managed, incase if installation fails. 
echo "------------------$PACKAGE_NAME:Install_fails-------------------------------------"
echo "$PACKAGE_URL $PACKAGE_NAME"
echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | GitHub | Fail |  Install_Fails"
exit 1

**Review by shubham-dayma-ibm — CHANGES_REQUESTED** [review]

File: m/multidict/multidict_ubi_9.3.sh (hunk 2 above)
Comment (aastha-sharma2) [reply m/multidict/multidict_ubi_9.3.sh]: > Don't see a case managed, incase if installation fails. echo "------------------$PACKAGE_NAME:Install_fails-------------------------------------" echo "$PACKAGE_URL $PACKAGE_NAME" echo "$PACKAGE_NAME | $PACKAGE_URL | $PACKAGE_VERSION | GitHub | Fail | Install_Fails" exit 1


Did changes as suggested and managed testcases in script.

**Review by shubham-dayma-ibm — APPROVED** [review]

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**